bob-common/src/data.rs

```diff
@@ -234,15 +235,15 @@ impl BobOptions {
 #[derive(Debug, Clone)]
 pub struct VDisk {
     id: VDiskId,
-    replicas: Vec<NodeDisk>,
+    replicas: HashMap<String, Vec<NodeDisk>>,
```

Comment:
I ran simple performance tests. Unfortunately, `HashMap` is slower than a vector when the number of elements is small (HashMap becomes faster with 10 elements or more in my tests). The usual number of replicas for Bob is 2 or 3. So this only reduces performance.
Unfortunately, this change must be reverted. Also please add a comment explaining why `Vec` is faster here